docs-v2/guides/self-hosting/enterprise-self-hosting.mdx

```diff
@@ -4,12 +4,83 @@ sidebarTitle: 'Enterprise self-hosting'
 description: 'Overview of the Enterprise self-hosted edition.'
 ---
 
+Nango offers an Enterprise Self-Hosted deployment option for customers who need full control over their infrastructure. This guide covers deployment, scaling, updates, and other key operational details.
+
+## What features are available?
+
 All paid features available on Nango Cloud are also available on the Enterprise self-hosted edition. 
 
+## What plan is needed?
+
 The Enterprise self-hosted edition requires a [Scale plan](https://www.nango.dev/pricing) subscription.
 
+## Who is this option for?
+
 Enterprise self-hosting is optimized for large, regulated enterprises.
 
+## How is it deployed? 
+
+By default, using Helm charts. There is no CloudFormation template or AWS CDK script provided at this time.
+
+We can also guide you to do custom deployments, e.g. for ECS. 
+
+## How and how often is it updated?
+
+Updates are distributed through versioned Helm charts.
+
+A staging environment is recommended to mitigate risks before applying updates to production.
+
+The update cadence is custom based on your need to benefit from new functionalities.
+
+## What cloud provider does Nango support? 
+
+All major cloud providers are supported (AWS, GCP, Azure).
+
+## What is the recommended configuration? 
+
+- All services (Server, Persist, Runner, Jobs, Orchestrator): 1 CPU, 2GB RAM per service
+- Postgres: 2 CPU, 8GB RAM, 128GB storage
+- Redis: 128MB
+- ElasticSearch: 8.5 vCPU, 1GB RAM, 30GB storage
```

Comment:
I think 8.5 vCPU is just a max available in that plan, but truthfully it's not needed to have that much, 2 vCPU should be fine